src/validations/access/subgraph/scenes.ts

```diff
@@ -51,6 +52,9 @@ export function createSceneValidateFn({
   const logger = logs.getLogger('scenes access validator')
 
   const SCENE_LOOKBACK_TIME = ms('5m')
+  const SCENE_VALIDATIONS_CONCURRENCY = process.env.SCENE_VALIDATIONS_CONCURRENCY
```

Comment:
I did it like this because is the way we are also resolving ADR's timestamps which also comes from environment. There is no config component in place right now on this library but I'm seeing `@well-known-components/env-config-provider` installed.